project_management/estimating.md

```diff
@@ -0,0 +1,65 @@
+# Estimating
+
+Estimates are key to what we do.
+
+We are in the business of selling software creation hours. We want to do this as inexpensively as possible while maintaining a healthy revenue for growth and downtimes. As a group, we need to hold appropriate tension between cost and quality.
+
+Estimates should be made by two developers at a minimum; estimates are shared, and ideally by the implementing team.
+When you win and lose together, your estimates get even better.
+
+## Derive duration
+
+We focus on size estimates (how much), not duration estimates (how long). Related, but useful independently.
+Estimates are a combination of effort, risk, complexity, etc. Separating size from duration enables agility in scheduling and iteration planning.
+
+## Split the work
+
+Before the work can be estimated, it has to be split into chunks of a manageable size. Depending on how reliable it needs to be (the estimate type), we will create user stories encompassing a larger amount of functionality or more detailed user stories that define a specific functionality. The team must focus on providing just enough details so that the client can make the right decision.
```

Comment:
```suggestion
Before the work can be estimated, it has to be split into chunks of a manageable size. Depending on how reliable it needs to be (the [estimate type](#different-estimate-types)), we will create user stories encompassing a larger amount of functionality or more detailed user stories that define a specific functionality. The team must focus on providing just enough details so that the client can make the right decision.
```